src/Illuminate/Mail/Mailer.php

```diff
@@ -112,6 +112,18 @@ public function alwaysFrom($address, $name = null)
 	}
 
 	/**
+	 * Set the global to address and name.
+	 *
+	 * @param  string  $address
+	 * @param  string  $name
```

Comment:
@GrahamCampbell Parameters that default to null are documented like that in many places in the framework. Just one example: [Query Builder](https://github.com/laravel/framework/blob/5.0/src/Illuminate/Database/Query/Builder.php#L312-L323)